### ISSUE TITLE
Profile cookie as well?

### ISSUE THREAD

**Issue by pbarney** [lhall-adexos/xdebug-ext#1]
Other (pre-Quantum) xdebug add-ons allowed setting of XDEBUG_SESSION, XDEBUG_PROFILE and even XDEBUG_TRACE. Do you have any thoughts about adding this?

Thanks for at least making XDEBUG work on Quantum!

**Comment by lhall-amphibee** [comment]
It didn't seem there was a lot of demand for those features, but I can look into it, yes.

I still want to keep the extension super simple, so new icons will probably be added as an option in the settings.

Thanks for your feedback.

**Comment by lhall-amphibee** [comment]
A little update on this topic : WebExtension can't handle more than 1 icon per module. This is being worked on but there's no sign of a release anytime soon.

See : https://bugzilla.mozilla.org/show_bug.cgi?id=1215064

**Comment by pbarney** [comment]
Thanks for the update! Much appreciated.

Just a thought, but what about an icon that has a dropdown to enable SESSION, DEBUG and TRACE? The available options could be set in the options screen. Extra credit: If only one option is checked, then the button isn't a dropdown.

I would help implement this but I have never worked on an extension before.

**Comment by lhall-amphibee** [comment]
Yes, this could work. There's already some code for the new settings page : https://github.com/lhall-adexos/xdebug-ext/commits/feature/trace_profiling_support

I'm currently working on some bugs I found when using weird domains, so I'm focusing on that for now. Feel free to play around if you want, it's just HTML/CSS/JS :)
